src/Pages/About/About.jsx

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import star from '../../Logo/star.svg';
+import redLine from '../../Logo/redLine.svg';
+import styles from './About.module.css';
+import profile from '../../Logo/pk.svg';
+import social from '../../Logo/SocialIcons.svg';
 
 function About() {
   return (
-    <div>About</div>
-  )
+    <>
+       <div className={styles.bodyContainer}>
+        <h1>Behind the Vision, Crafting</h1>
+          <h1>Tomorrow's <span className={styles.UnderlineText}>Successes.<img className={styles.underLine} src={redLine} alt='GreenLine'/></span></h1>
+        <p>Get to know us behind the vision and the values that drive our commitment to excellence every day.</p>
+        <div className={styles.Container}>
+          <p>We are <span>product tinkers,</span></p>
+          <br></br>
+          <br></br>
+          <p>Since inception, things have changed quite a bit.</p>
+          <br></br>
```

Comment:
Instead of br tags, you could use stylings/css to give some margin/padding. But for now I think its okay. Anyway good job, keep it up! Vakunde

Reply:
br le sajilo laagyo

Reply:
Ok, its fine for now could be a thing to note down for later, perhaps!